data/timeline.json.unvalidated.in

```diff
@@ -958,34 +957,788 @@
+                        "chat::ada::processing::wave_thickness",
+                        "chat::ada::processing::wave_thickness_pause"
+                     ],
+                    "no": [
+                        "chat::ada::processing::wave_stroke_sample_code_example",
+                        "chat::ada::processing::wave_arcs_num",
+                        "chat::ada::processing::wave_thickness",
+                        "chat::ada::processing::wave_thickness_pause"
+                    ]
+                }
+            }
+        },
+        {
+            "name": "chat::ada::processing::wave_stroke_sample_code_example",
+            "type": "chat-actor-attachment",
+            "data": {
+                "actor": "Ada",
+                "attachment": {
+                    "desc": "Wave 30",
+                    "path": "wave_30/wave_30.pde"
+                }
+            }
+        },
+        {
+          "name": "chat::ada::processing::wave_arcs_num",
+          "type": "chat-actor",
+          "data": {
+            "actor": "Ada",
+            "message": "That’s much more exciting! With the “Int” command (short for “integer,” we’ve changed the number of arcs that form our wave. "
+          }
+        },
+        {
+          "name": "chat::ada::processing::wave_thickness",
+          "type": "chat-actor",
+          "data": {
+            "actor": "Ada",
+            "message": "Now let’s have some fun with the thickness of the arcs. On line 6, let’s change the strokeWeight from 5 to 25."
+          }
+        },
+        {
+            "name": "chat::ada::processing::wave_thickness_pause",
+            "type": "input-user",
+            "data": {
+                "actor": "Ada",
+                "input": {
+                    "type": "choice",
+                    "settings": {
+                        "prompt": "Ready to continue?",
+                        "choices": {
+                            "yes": {
+                                "text": "Ready!"
+                            }
+                        }
+                    }
+                },
+                "responses": {
+                    "yes": [
+                        "chat::ada::processing::wave_colors",
+                        "chat::ada::processing::wave_stroke",
+                        "chat::ada::processing::wave_stroke_pause"
+                    ]
+                }
+            }
+        },
+        {
+          "name": "chat::ada::processing::wave_colors",
+          "type": "chat-actor",
+          "data": {
+            "actor": "Ada",
+            "message": "Look at those crazy arcs! Let’s drop it down to 10 and have some fun with colors."
+          }
+        },
+        {
+          "name": "chat::ada::processing::wave_stroke",
+          "type": "chat-actor",
+          "data": {
+            "actor": "Ada",
+            "message": "Look for the “stroke” function, which currently has a variable of (255)."
```

Comment:
"Has a variable of 255" seems kinda misleading. Perhaps, "is currently being being run with 255"